spec/javascript/controllers/internal_events_controller_spec.js

```diff
@@ -0,0 +1,69 @@
+import InternalEventsController from 'internal_events_controller.js';
+
+describe('InternalEventsController', () => {
+  let controller;
+
+  beforeEach(() => {
+    controller = new InternalEventsController();
```

Comment:
I would replace them or you'll end up doubling up; if you test the output of the controller actions by asserting against the HTML it will also let you refactor your methods without needing to update your tests, which is beneficial